### PR TITLE
updated caption on my md slide

### PR DESCRIPTION
**Pull request by robertefreeman**
blah blah blah

#4 

List of things I completed:
- Added a caption to slide

cc: @DODTeam1

### CODE REVIEW

**Review by i-marsh — APPROVED** [review]
I think 5 legs is right :)

File: _posts/0000-01-03-robertefreeman.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+### Why does Robotocat only have 5 legs?
```
Comment (beardofedu) [inline _posts/0000-01-03-robertefreeman.md:6]: i ask myself the same question all the time.

File: _posts/0000-01-03-robertefreeman.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-03-robertefreeman.md:3]:
```suggestion
title: "Welcome to Robert's slide!"
```

**Review by beardofedu — APPROVED** [review]
looks good to me 🚢

**Pull request merged**